include/micm/util/cuda_vector_matrix.hpp

```diff
@@ -1,7 +1,7 @@
 #include <micm/util/cuda_vector_matrix.cuh>
 #include <micm/util/vector_matrix.hpp>
 #include <type_traits>
-
+#include <iostream>
```

Comment:
is this still needed?

Reply:
I thought I had removed all the debug code but I forgot the header file. Removed.